src/test/java/org/cactoos/scalar/ThrowsOnFalseTest.java

```diff
@@ -24,50 +24,38 @@
 
 package org.cactoos.scalar;
 
-import org.hamcrest.MatcherAssert;
-import org.hamcrest.Matchers;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
+import org.hamcrest.core.IsEqual;
+import org.junit.Test;
+import org.llorllale.cactoos.matchers.Assertion;
 
 /**
  * Test suite for {@link ThrowsOnFalse}.
  *
  * @since 0.56.0
  */
-final class ThrowsOnFalseTest {
+public final class ThrowsOnFalseTest {
 
-    @Test
-    void throwsOnFalse() {
-        final String msg = "test message";
-        final String message =
-            Assertions.assertThrows(
-                IllegalArgumentException.class,
-                () -> new ThrowsOnFalse(
-                    () -> false, msg
-                ).value()
-            ).getMessage();
-        MatcherAssert.assertThat(
-            "Exception message in right format",
-            message,
-            Matchers.equalTo(msg)
-        );
+    @Test(expected = IllegalArgumentException.class)
+    public void throwsOnFalse() throws Exception {
+        new ThrowsOnFalse(
```

Comment:
@h1alexbel please use `org.llorllale.cactoos.matchers.Throws` with `Assertion` to check for exception (instead of using `expected = IllegalArgumentException.class`)